modules/holiday-snow.php

```diff
@@ -31,16 +31,44 @@ public function blog_field_html() {
 	}
 
 	public function holiday_snow_option_updated() {
+		
+		/**
+		 * Fires when the holiday snow option is updated.
+		 * 
+		 * @since 2.0.3
+		 */
 		do_action( 'jetpack_holiday_snow_option_updated' );
 	}
 }
 
 function jetpack_holiday_snow_script() {
+	
+	/**
+	 * Filter the holiday snow.
+	 * 
+	 * There's no actual "chance" for snow, it's either on or off.
```

Comment:
That's true, but we wouldn't want a dev to think that by setting that filter to `true`, they'd force the snow to appear regardless of every other setting or date format.

How about "Can the snow be displayed?"